test/lib/utils/source-code-utils.js

```diff
@@ -42,6 +42,31 @@ describe ('Testing SourceCode instance for exposed functionality', function () {
 		sourceCodeObject.should.have.property ('getParent');
 		sourceCodeObject.getParent.should.be.type ('function');
 
+		// functions for node type checking
```

Comment:
Please add cases to test the functionality of your util methods too. You need only have 2 cases per method:
1 that is expected to return `true` & another expected to return `false`.